documentation/docs/pages/operators/nodes/nym-node/bonding.mdx

```diff
@@ -0,0 +1,100 @@
+```sh
+echo "$(curl -4 https://ifconfig.me)"
+```
+
+###### 2. Bond to correct ports
+
+- In your wallet: Open the box called `Show advanced options` and make sure all your ports are set correctly, like the values in this table:
+
+| Node type              | Port name                           | Correct port value |
+| :--                    | :--                                 | :--                |
+| Mixnode                | Mix port                            | `1789`             |
+| Mixnode                | Verloc port                         | `1790`             |
+| Mixnode                | HTTP api port (picture below)       | `8080`             |
+| Gateway (entry & exit) | Mix port                            | `1789`             |
+| Gateway (entry & exit) | Client WS API port                  | `9000`             |
+
+- If you bonding `nym-node --mode mixnode` through *Bond mixnode* desktop wallet menu, change manually *HTTP api port* value from deprecated `8000` to `8080` - a generic `nym-node` HTTP port (for all modes).
+
+![](/images/operators/wallet-screenshots/new_http_port.png)
+
+###### 3. Enter your values and sign with your node
+
+- Enter the `Amount`, `Operating cost` and `Profit margin` and press `Next`
+
+<Callout color="danger" emoji="⚠️">
+
+If you are part of [Nym Delegation Program](https://delegations.explorenym.net) or Service Grants Program, make sure your values are within the [rules](https://forum.nymtech.net/t/nym-delegations-program-update/466) of the programs. Operators setting up larger OP or PM than defined in the rules will be excluded from the program without prior warning!
+</Callout>
+
+- You will be asked to run a `sign` command with your `nym-node` - copy and paste the long signature as the value of `--contract-msg` and sing it on your VPS:
+
+```sh
+./nym-node sign --contract-msg <PAYLOAD_GENERATED_BY_THE_WALLET>
+```
+
+- Copy the resulting signature string and paste it into the wallet nodal, press `Next` and confirm the transaction:
+
+```sh
+# This is just an example, copy the one from your process
+>>> The base58-encoded signature is:
+2bbDJSmSo9r9qdamTNygY297nQTVRyQaxXURuomVcRd7EvG9oEC8uW8fvZZYnDeeC9iWyG9mAbX2K8rWEAxZBro1
+```
+
+![Paste Signature](/images/operators/wallet-screenshots/wallet-sign.png)
+
+*This image is just an example, copy-paste your own base58-encoded signature*
+
+</Steps>
+
+Your node will now be bonded and ready to receive traffic, latest at the beginning of the next epoch (at most 1 hour).
+
+
+If everything worked, you'll see your node running on the either the [Sandbox testnet network explorer](https://sandbox-explorer.nymtech.net) or the [mainnet network explorer](https://explorer.nymtech.net), depending on which environment you're running.
+
+
+## Bond via the CLI (power users)
+
+If you want to bond your Mix Node via the CLI, then check out the [relevant section in the Nym CLI](https://nymtech.net/docs/tools/nym-cli.html#bond-a-mix-node) docs.
```

Comment:
need to fix link to local page

Reply:
all of the urls pointing to developers or networks or sdks are to be done post merge.

I will grep them